Cargo.toml

```diff
@@ -45,12 +45,15 @@ dirs = "5.0.1"
 indoc = { version = "2.0.4", optional = true }
 log = { version = "0.4.8", default-features = false }
 miette = { version = "5.7.0", features = ["fancy"] }
+notify = { version = "8.1.0", optional = true }
```

Comment:
From the docs of `notify-debounce-mini` it seems `notify` is not needed unless you are overriding the default features, which currently the code is not doing. Let's remove this line and exclusively re-export from `notify-debouncer-mini`.